portfolio/src/main/webapp/script.js

```diff
@@ -48,8 +49,74 @@ function closeModal() {
 // Request content from server
 function getServerData() {
   console.log("Fetching data.");
-  const promise = fetch("/data").then(response => response.json()).then(greeting => {
-    console.log(greeting);
-    document.getElementById('data-container').innerText = greeting;
-  });
-}
\ No newline at end of file
+  const maxElement = document.getElementById("quantity");
+  const maxToDisplay = maxElement.value;
+  const sort = document.getElementById("sort").value;
+  const img = document.getElementById("modal-img").src;
+
+  fetch("/comment?quantity=" + maxToDisplay + "&sort=" + sort + "&img=" + img).then(response => response.json()).then(jsonObj => {
+    const comments = jsonObj.comments;
+    console.log(comments[comments.length - 1]);
```

Comment:
You don't have to remove all these logging statement, but if u were to push code to production we would remove all logging statements

Reply:
Oh yeah that makes sense!